examples/sdk-examples/src/examples/execution/RawExecuteSrc.tsx

```diff
@@ -58,8 +58,8 @@ const RawExecuteExample: React.FC = () => {
                                 <li key={locationState}>
                                     <strong>{locationState}</strong>
                                     <ul>
-                                        {_locations.map((location) => (
-                                            <li key={location as string}>{location}</li>
+                                        {_locations.map((location: any) => (
```

Comment:
No idea how to fix this one .. was fighting with it already during typescript update